server/database/queries/interests/getInterests.js

```diff
@@ -2,7 +2,7 @@ const dbConnection = require('../../config/dbConnection');
 
 exports.getInterests = () => {
   const sql = {
-    text: '',
+    text: 'SELECT * FROM interests',
     values: [],
```

Comment:
```suggestion
    
```
if there are no values on your query then remove the values and you can refactor your code 
so you can write it like this 
```js
exports.getInterests = () => {
  const sql = 'SELECT * FROM interests';
  return dbConnection.query(sql);
};

```